### DISCUSSION TITLE
How to customize

### DISCUSSION THREAD

**Discussion by tommoholmes** [saleor/saleor#10220] [Q&A]
I want to customize product, but when I search, its suggest to use metadata,
but when I check in graphql,
metadata not appear
https://docs.saleor.io/docs/3.x/developer/api-reference/inputs/product-create-input

**Comment by karolkielecki** [comment]
Hi @tommoholmes, could you please provide more info? How would you like to customize products?

**Reply by tommoholmes** [reply]
I want to create product with mutation addProduct,
and added some metadata
